package.json

```diff
@@ -46,7 +46,7 @@
     "setup": "npm i  && npm i --no-save mongodb@2.2.27",
     "test": "./scripts/test.sh",
     "test:docker:keep": "cd docker && docker-compose run test",
-    "test:docker": "npm run test:docker:keep; cd docker && docker-compose down",
+   "test:docker": "npm run test:docker:keep; cd docker && docker-compose down",
```

Comment:
why

Reply:
Not the slightest idea. Re-adding the space.